diff --git a/index.js b/index.js
index 0e8668d62c65184423e0b89269090d5cd3ebdbcc..f78d47b50f7059f6024435abc396d4da3198e0f9 100644
--- a/index.js
+++ b/index.js
@@ -1449,7 +1449,7 @@ const GraphQLURL =
 function patchBigInt() {
     if (!BigInt.prototype.toJSON) {
         BigInt.prototype.toJSON = function () {
-            return Number(this);
+            return String(this);
         };
     }
 }
diff --git a/index.mjs b/index.mjs
index f874ea02c1392eac0bff978e8a1daf253f37dd31..472ead89db463779da98b0ca4f680ece5d7792d6 100644
--- a/index.mjs
+++ b/index.mjs
@@ -1445,7 +1445,7 @@ const GraphQLURL =
 function patchBigInt() {
     if (!BigInt.prototype.toJSON) {
         BigInt.prototype.toJSON = function () {
-            return Number(this);
+            return String(this);
         };
     }
 }
