diff --git a/lib/generator/config.js b/lib/generator/config.js
index 985f338be23aa3526e4ab2f131ae9e95585daafa..f12073e5d500a363a381bf917af26458a9262c7f 100644
--- a/lib/generator/config.js
+++ b/lib/generator/config.js
@@ -10,13 +10,13 @@ exports.supportedQueryActions = [
     "groupBy",
 ];
 exports.supportedMutationActions = [
-    "create",
-    "createMany",
-    "delete",
-    "update",
-    "deleteMany",
-    "updateMany",
-    "upsert",
+    // "create",
+    // "createMany",
+    // "delete",
+    // "update",
+    // "deleteMany",
+    // "updateMany",
+    // "upsert",
 ];
 exports.modelsFolderName = "models";
 exports.enumsFolderName = "enums";
